Automattic/Gravatar-SDK-iOS
# Alt text editor height

Closes #

### Description

This PR allows the Alt Text editor sheet to have a height close to its content. Avoiding it being full screen.

We decided to not use intrinsic content height due to its complexity. A constant height inside a scroll view will do for now.

![CleanShot 2024-12-18 at 13 37 20](https://github.com/user-attachments/assets/4e6f47f2-ea83-4812-88cf-0ce5f2a7eff6)

### Testing Steps

- Open the QuickEditor.
- Tap on the contextual menu of an avatar
- Choose Alt Text
  - Check that the alt text editor height is not full screen. 
  - Try different font sizes, landscape, scroll to dismiss, or anything else which could affect the sheet size.
